.travis.yml

```diff
@@ -3,13 +3,23 @@ language: php
 matrix:
     include:
         - php: 5.6
+          env: LCOBUCCI_JWT_VERSION=^3.4
```

Comment:
I guess this isn't even necessary because version 4 requires PHP 7.4 and it will install 3.x anyway but I assume it also doesn't hurt to be explicit about it.

Reply:
I did it that way, since I had no clue how to do line 54 without it. By the way: I'm also not sure if this works well for the hhvm-builds, but they are failing in an earlier stage at the composer self-update. I don't know how to fix that and at least it's not related to this issue/branch.